src/frontends/onnx/frontend/src/ops_bridge.cpp

```diff
@@ -293,12 +312,18 @@ static const char* const PYTORCH_ATEN_DOMAIN = "org.pytorch.aten";
     m_map[domain_][name_].emplace(ver_, std::bind(op::set_##ver_::fn_, std::placeholders::_1));
 
 OperatorsBridge::OperatorsBridge() {
-    REGISTER_OPERATOR("Abs", 1, abs);
-    REGISTER_OPERATOR("Acos", 1, acos);
-    REGISTER_OPERATOR("Acosh", 1, acosh);
-    REGISTER_OPERATOR("Add", 1, add);
-    REGISTER_OPERATOR("Add", 7, add);
-    REGISTER_OPERATOR("And", 1, logical_and);
+    register_operator("Abs", VersionRange{1, 5}, op::set_1::abs, "Legacy consumed_inputs is not supported");
+    register_operator("Abs", VersionRange::from_version(6), op::set_6::abs);
+    register_operator("Acos", VersionRange::single_version_for_all_opsets(), op::set_7::acos);
+    register_operator("Acosh", VersionRange::single_version_for_all_opsets(), op::set_9::acosh);
+    register_operator("Add", VersionRange{1, 5}, op::set_1::add, "Legacy consumed_inputs is not supported");
+    register_operator("Add", VersionRange::in_version(6), op::set_6::add);
+    register_operator("Add", VersionRange{7, 12}, op::set_7::add);
+    register_operator("Add", VersionRange::in_version(13), op::set_13::add);
+    register_operator("Add", VersionRange::from_version(14), op::set_14::add);
```

Comment:
It's a great example to demonstrate all of the VersionRange possibilities, but why not just register the same Add by `from_version(7)`?
```suggestion
    register_operator("Add", VersionRange{1, 5}, op::set_1::add, "Legacy consumed_inputs is not supported");
    register_operator("Add", VersionRange::in_version(6), op::set_6::add);
    register_operator("Add", VersionRange::from_version(7), op::set_7::add);
```

Reply:
I've thought about it, but my purpose was to do it in the most explicitly way - it means that all opset versions where op has been changed (for Add - 1,6,7,13,14) are expilictly listed during registration. On the other hand, you are right that changes from versions 13 and 14 are not significant.
I haven't a very strong opinion here - we should balance between number of lines vs unification with spec.
@tomdol What do you think?

Reply:
Agree, let's keep it as is